core/server/users.js

```diff
@@ -18,7 +18,11 @@ Meteor.publish('users', function (levelId) {
     };
   }
 
-  return Meteor.users.find(filters, { fields: mainFields });
+  const users = Meteor.users.find(filters, { fields: mainFields });
+  const guildIds = users.map(u => u.guildId).filter(Boolean);
+  const guilds = Guilds.find({ _id: { $in: [...new Set(guildIds)] } });
+
+  return [users, guilds];
```

Comment:
I was wondering, is this ok in terms of performance? I haven't tested amplitude so maybe you have some info about it